deploy/charts/emqx-enterprise/templates/service.yaml

```diff
@@ -38,7 +38,7 @@ spec:
     {{- else if eq .Values.service.type "ClusterIP" }}
     nodePort: null
     {{- end }}
-    {{- if not (empty .Values.emqxConfig.EMQX_LISTENERS__TCP__INTERNAL__BIND) }}
+    {{- if not (empty .Values.emqxConfig.EMQX_LISTENERS__TCP__DEFAULT) }}
```

Comment:
not sure if this is correct change, but it's how it is in `emqx`

Reply:
the `emqx` is wrong, should use `EMQX_LISTENERS__TCP__INTERNAL__BIND`